packages/dashboard/src/components/layout-manager/layout-manager.tsx

```diff
@@ -0,0 +1,48 @@
+import { Mosaic, MosaicWindow, MosaicBranch } from 'react-mosaic-component';
+import 'react-mosaic-component/react-mosaic-component.css';
+import './layout-manager.css';
+import Debug from 'debug';
+import React from 'react';
+import BuildingMapWindow from './building-map-window';
+
+const debug = Debug('LayoutManager');
+
+export default function LayoutManager(_props: {}): React.ReactElement {
```

Comment:
I'm not familiar with mosaic but it seems like the layout in this component is fixed? I would think `LayoutManager` would be something that works with different layouts.

Reply:
What confuses me is this line
```tsx
if (id === 'map_2d') return <BuildingMapWindow path={path} />;
```
it seems hardcoded to return a `BuildingMapWindow` or a `div`. It's not clear to me how to use this component with different windows/panels. I thought following react pattern, it should take in some props and childrens to control what windows are rendered.

Reply:
In that case, I think we can show it using storybook, `LayoutManager` could be a story, and there would be no need to add a wip route to the dashboard.